### ISSUE TITLE
Ude, wenn?

### ISSUE THREAD

**Issue by harveyspecter93** [RoadJDK/EnergyAir-Bot-2022#1]
Wär schön, wed mau öppis miechsch :)

**Comment by ThePrinoob** [comment]
haut nümeno au stund mitmache, bringt fasch nüt e bot mache oder

**Comment by RoadJDK** [comment]
es het e Möglichkeit gä, API calls zspamme und somit hetme i 5 minute tickets gwunne. di lücke isch aber scho gschlosse worde. ig bi witerhin uf dr suechi nachemne wäg.
zeifachschte wär eifach dr pc deheime loufe zla und e bot würd für di dfrage beantworte (jedoch chaner nume schetzä, da mir ke date mit aune frage hei)

**Comment by ThePrinoob** [comment]
ah was ds wär nice gsi..
du chasch via API dlösig hole
https://energy.ch/api/quiz/question
questionId isch immer glich wie die richtigi option Id

**Comment by TimH04** [comment]
Theoretisch chönti mer per Account Generator mehreri Accounts ha, weli nachher immer e dem Intervall d froge löse

**Comment by harveyspecter93** [comment]
I hamer miteem API spame o 8 Tickets chönne hole xD 
Momentan probierenis mit random Session-Ids. Bis ize aber erfolglos.. isch aber o z'erwarte bi derä Azahl vo mögleche Ids ^^ 
Cheers

**Comment by harveyspecter93** [comment]
> Theoretisch chönti mer per Account Generator mehreri Accounts ha, weli nachher immer e dem Intervall d froge löse

Jaein. D E-Mail muess IMHO zersch verifiziert/bestätigt wärde, bevor mä cha teilneh.

**Comment by TimH04** [comment]
> > Theoretisch chönti mer per Account Generator mehreri Accounts ha, weli nachher immer e dem Intervall d froge löse
> 
> Jaein. D E-Mail muess IMHO zersch verifiziert/bestätigt wärde, bevor mä cha teilneh. 

Jo, mir chönti doch aber per Nodemailer d mail "abfange" und mit puppeteer de Link öffne oder klappt das so ned?

**Comment by RoadJDK** [comment]
> > Theoretisch chönti mer per Account Generator mehreri Accounts ha, weli nachher immer e dem Intervall d froge löse
> 
> Jaein. D E-Mail muess IMHO zersch verifiziert/bestätigt wärde, bevor mä cha teilneh. 

das isch kes problem

**Comment by TimH04** [comment]
> I hamer miteem API spame o 8 Tickets chönne hole xD 
> Momentan probierenis mit random Session-Ids. Bis ize aber erfolglos.. isch aber o z'erwarte bi derä Azahl vo mögleche Ids ^^ 
> Cheers

Uf welem API Endpoint hesch das chöne mache?

**Comment by harveyspecter93** [comment]
> Uf welem API Endpoint hesch das chöne mache?

Die 8 Tix? Direkt ufem /api/quiz/answer. Isch aber no so gsi, dass mä ä random userId het chönne mitgä im body.

**Comment by RoadJDK** [comment]
Hüt wird e commit uf das Repo cho. stay tuned

**Comment by TimH04** [comment]
Hät Energy System mit de Question ID und de Lösig ID geändert? 
![image](https://user-images.githubusercontent.com/48881103/188104103-7c6e055b-d3a2-49c2-96ea-14a7f121e920.png)

**Comment by CindyJacks** [comment]
Wie sehts us? Funktioniert de bot? Händr scho tickets gwunne? Danke für dini arbeit :)

**Comment by TimH04** [comment]
De Bot klappt nüme. Sie händ d IDs apasst. Mer chönti halt probiere s Resultat 1 mal azwähle und dann botte mit mehrere Accounts. 
Falls normal Tickets gwüne wilsch, Metaverse Gewinnspiel sind relativ eifach zum gwüne... letzte fritig hesch z.B müesse uf s Dach bem Energy Studio go und die erste 5 händ Tickets becho

**Comment by RoadJDK** [comment]
ihr müesst läsä lerne

!! This is already fixed again. I will make the new bot public when me and my friends all got tickets !!

I hane funktionierende, aber no ke tickets. Weni ha ladis ufe.

Weder dlogik wüsst chöiterne säuber apasse
